students/msemerni/hw-4/js/index.js

```diff
@@ -0,0 +1,54 @@
+//// Task 1
+//// Створити функцію для підрахунку кількості голосних англійського алфавіту (a, o, e, i, u, y) в довільному тексті.
+//// Деталі: 
+//// функція повинна отримувати 1 аргумент (текст для пошуку (string)) 
+//// та повертати в консоль кількість голосних, якщо такі є в тексті;
+
+function searchVowels(string) {
+  string = string.toLowerCase();
```

Comment:
Please do not reassign function arguments. Create a new variable with modified value instead.